src/xcode/ENA/ENA/Source/Models/Profile/AntigenTestProfile.swift

```diff
@@ -91,5 +91,8 @@ struct AntigenTestProfile: Codable {
 	var city: String?
 	var phoneNumber: String?
 	var email: String?
-
+	
+	var fullName: String {
+		return (firstName ?? "") + " " + (lastName ?? "")
```

Comment:
Just a suggestion:
```suggestion
		[firstName, lastName]
			.compactMap { $0 }
			.joined(separator: " ")
```
(With this if none is set the result is "" instead of " ")